src/js/Boards/PlayBoard.js

```diff
@@ -0,0 +1,18 @@
+import {Board} from './Board.js';
+
+export class PlayBoard extends Board{
+
+    constructor(boardId, boardSize, elem, tblCellsElemArr){
+        super(boardId, boardSize, elem, tblCellsElemArr);
+
+         //this.boardElem.addEventListener("click", eveHan); // TODO: Find how to assign to obj elem
+    }
+
+    // Checks if the cell is empty, so we can put game sign in it
+    isEmptyCell(row, col){
+        if (this.getCellValue(row, col) === null){
```

Comment:
can't you just return the comparison result as a boolean, e.g.

"return this.getCellValue(row, col) === null" ?

Also, one liner functions can be implemented using arrow syntax IIRC